apps/dcos_net/test/dcos_net_mesos_listener_tests.erl

```diff
@@ -533,11 +565,21 @@ hello_overlay_setup() ->
 pod_tasks_setup() ->
     setup("pod-tasks.json").
 
+recovered_agents_setup() ->
+    setup("recovered-agents.json").
+
+setenv() ->
+    application:set_env(dcos_net, is_master, true),
+    application:set_env(dcos_net, mesos_agents_readiness_timeout,
+        timer:seconds(1)),
+    application:set_env(dcos_net, mesos_tasks_readiness_timeout,
+        timer:seconds(1)).
+
 setup(FileName) ->
     Lines = read_lines(FileName),
+    setenv(),
```

Comment:
If the application is not loaded `application:set_env/n` functions will do nothing. I guess there is a side effect and the application is already loaded by some other tests. Could you please set configuration after `application:load(dcos_net)`?